sql_scripts/answer_view.sql

```diff
@@ -1,8 +1,20 @@
+with t1 as (select user_id,
+    display_name,
+    answer_id,
+    question_id,
+    is_accepted,
+    score,
+    answer_date,
+    max(if(is_downvoted = 'true','true', 'false' )) over (partition by display_name) as with_downvote
```

Comment:
Instead of using `display_name` as a partition use an ID instead. Display name can have duplicates, eg. There are 2 people named Mark, John, etc.